src/overlays/actors/ovl_Fishing/z_fishing.c

```diff
@@ -4714,37 +4728,37 @@ void Fishing_DrawGroupFishes(PlayState* play) {
     CLOSE_DISPS(play->state.gfxCtx, "../z_fishing.c", 8099);
 }
 
-static u16 D_80B7AFB8[] = { 0x4096, 0x408D, 0x408E, 0x408F, 0x4094, 0x4095 };
+static u16 sPondOwnerTextIds[] = { 0x4096, 0x408D, 0x408E, 0x408F, 0x4094, 0x4095 };
 
 void Fishing_HandleOwnerDialog(Fishing* this, PlayState* play) {
-    switch (this->unk_15C) {
+    switch (this->state) {
         case 0:
-            if (D_80B7E0AC == 0) {
+            if (sFishingPlayingState == 0) {
                 if (sLinkAge != LINK_AGE_CHILD) {
-                    if ((HIGH_SCORE(HS_FISHING) & 0x100) && !(HIGH_SCORE(HS_FISHING) & 0x200)) {
-                        this->actor.textId = 0x4093;
+                    if ((HIGH_SCORE(HS_FISHING) & HS_FISH_PLAYED_CHILD) && !(HIGH_SCORE(HS_FISHING) & HS_FISH_PLAYED_ADULT)) {
```

Comment:
We use a formatter to format the code, among other things it enforces a line length limit

You can run it with `./format.py` at the repo's root